xo/expressions/comparable-path.ts

```diff
@@ -16,18 +16,36 @@
  * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
  */
 
```

Comment:
maybe we should rename this file since there's nothing like a "comparable path", anymore